apps/inkbeard/src/stores/featureFlags.ts

```diff
@@ -2,5 +2,9 @@ import { ref } from 'vue';
 import { defineStore } from 'pinia';
 
 export const useFeatureFlagStore = defineStore('featureFlags', () => (
-  { flags: ref({}) }
+  {
+    flags: ref({
+      examples: { enabled: false },
```

Comment:
This needs to be set up here first, otherwise typescript will complain this doesn't exist.